gcloud/orb.yaml

```diff
@@ -122,7 +122,7 @@ commands:
           name: install dependencies
           command: |
             if which apk >/dev/null; then
-              apk add --no-cache --no-progress bash curl python3
+              apk add --no-cache --no-progress bash curl curl-dev python3
```

Comment:
Do we want to still keep `curl` along with `curl-dev`?

Reply:
`curl-dev` didnt install `curl` when I tried